packer.json

```diff
@@ -61,6 +61,7 @@
     "salt_path": "{{ env `SALT_PATH` }}",
     "pyzmq_version": "{{ env `PYZMQ_VERSION` }}",
     "aws_instance_type": "t3.2xlarge",
+    "aws_spot_price": "0.3328",
```

Comment:
Can we switch on spot without pricing?

Reply:
According to the documentation we should be able to, but when I tried it tried to start a normal instance and could not find its type - this was the only working spot scenario I could find.

Reply:
Maybe we have an old version of packer?

Reply:
Tried it with the newest version and the same happened - so I rolled back the version change